deploy/addons/freshpod/freshpod-rc.yaml

```diff
@@ -0,0 +1,47 @@
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: freshpod
+  namespace: kube-system
+  labels:
+    k8s-app: freshpod
+    kubernetes.io/minikube-addons: freshpod
+    addonmanager.kubernetes.io/mode: Reconcile
+spec:
+  replicas: 1
+  selector:
+    k8s-app: freshpod
+    addonmanager.kubernetes.io/mode: Reconcile
+  template:
+    metadata:
+      labels:
```

Comment:
do we need `kubernetes.io/minikube-addons: freshpod` here as well, so that the Pods get this label, too? [The documentation](kubernetes.io/minikube-addons: freshpod) is unclear.